src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.form__wrapper {
+  width: 1009px;
+  margin: 20px auto;
+}
+
+.big__bar {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 26px 25px;
+  padding-left: 62px;
+  border: 1px solid rgba(100, 100, 100, 0.2);
+  width: 100%;
+  height: 70px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small__bar {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 13px 15px;
+  padding-left: 33px;
+  border: 1px solid rgba(100, 100, 100, 0.2);
+  width: 100%;
+  height: 42px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
as I mentioned above, pay attentiont to [checklist](https://github.com/Andriy-Shymkiv/layout_search-bar-airbnb/blob/develop/checklist.md) rule number 2, because here you have some simillar styles for both `big__bar` and `small__bar` blocks, so you can create a base class, for ex. `input`, and give to it repeating styles for both inputs.